### PR TITLE
Add March 2021 RubyGems Updates

### PR DESCRIPTION
**Pull request by lauragift21**
Hi, @deivid-rodriguez and @sonalkr132 can you help review the monthly updatE for March and also let me know if there's any additional update I need to add.

Thanks!

### CODE REVIEW

File: _posts/2021-04-05-march-rubygems-update.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## RubyGems News
+
+In March, we released new versions for [RubyGems(v3.2.15)](https://github.com/rubygems/rubygems/releases/tag/v3.2.15) and [Bundler(2.2.15)](https://github.com/rubygems/rubygems/releases/tag/bundler-v2.2.15) and worked on making Git sources faster and more disk efficient.
+
+For this Month, [Rubygems](https://github.com/rubygems/rubygems) gained [156 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-03-01%7D...master@%7B2021-03-31%7D), contributed by 12 authors. There were 2337 additions and 753 deletions across 111 files.
+
+## RubyGems.org News
+
+This month in RubyGems.org, we [enabled Rails v6.0 defaults](https://github.com/rubygems/rubygems.org/pull/2674 ) and [deployed update to Rails 6.1.3](https://github.com/rubygems/rubygems.org/pull/2675). We investigated and locked RubyGems.org accounts where the user's email domain had expired.
```
Comment (sonalkr132) [inline _posts/2021-04-05-march-rubygems-update.md:18]: > We investigated and locked RubyGems.org accounts where the user's email domain had expired. 

please remove this part.

**Comment by deivid-rodriguez** [comment]
Hi @lauragift21!

During March we actually released bundler from 2.2.12 to 2.2.15, and rubygems from 3.2.12 to 3.2.15. Have a look at https://rubygems.org/gems/bundler/versions and https://rubygems.org/gems/rubygems-update/versions.

As a recap of the work included in those releases, I think we can say that we mainly worked on making the client tools more secure, and making git sources faster and more disk efficient.

Thanks!

File: _posts/2021-04-05-march-rubygems-update.md (hunk 1 above)
Comment (sonalkr132) [inline _posts/2021-04-05-march-rubygems-update.md:18]: > (https://github.com/rubygems/rubygems.org/pull/2674 )

has space.

Comment (lauragift21) [reply _posts/2021-04-05-march-rubygems-update.md:18]: thanks fixed.

**Comment by lauragift21** [comment]
Great thanks! @deivid-rodriguez @sonalkr132

**Review by sonalkr132 — APPROVED** [review]

File: _posts/2021-04-05-march-rubygems-update.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## RubyGems.org News
+
+This month in RubyGems.org, we [enabled Rails v6.0 defaults](https://github.com/rubygems/rubygems.org/pull/2674) and [deployed update to Rails 6.1.3](https://github.com/rubygems/rubygems.org/pull/2675).
+
+RubyGems.org saw several bug fixes and updates this month, some of which include the following:
+
+- fixed upload of test coverage report to codeclimate -  [#2673](https://github.com/rubygems/rubygems.org/pull/2673)
+- made users emails private by default and updated existing accounts to hide emails - [#2663](https://github.com/rubygems/rubygems.org/pull/2663)
+- added text-only versions of emails to support more email clients - [#2652](https://github.com/rubygems/rubygems.org/pull/2652)
+- upgraded ES instance type and storage and added a strict rate limit` on the Search API endpoint. - [#2665](https://github.com/rubygems/rubygems.org/pull/2665)
```
Comment (deivid-rodriguez) [inline _posts/2021-04-05-march-rubygems-update.md:25]:
```suggestion
- upgraded ES instance type and storage and added a strict rate limit on the Search API endpoint. - [#2665](https://github.com/rubygems/rubygems.org/pull/2665)
```

**Review by deivid-rodriguez — APPROVED** [review]

**Pull request merged**